server/web_ui/user/src/views/apps.rs

```diff
@@ -121,14 +123,30 @@ impl AppsApp {
                             name,
                             display_name,
                             redirect_url,
-                            icon: _,
+                            has_image,
                         } => {
                             let redirect_url = redirect_url.to_string();
+                            let node_ref = NodeRef::default();
+                            if *has_image {
+                                let node_ref = node_ref.clone();
+                                let name = name.clone();
+                                spawn_local(async move {
+                                    let (_, status, blob, _) =
+                                        get_blob(format!("/ui/images/oauth2/{name}").as_str()).await.unwrap();
```

Comment:
Theres a server uri handler for this?

Reply:
yep https://github.com/ToxicMushroom/kanidm/blob/8175253bae820bd9584ba808a9bc8033c993b2f5/server/core/src/https/oauth2.rs#L96